scripts/generate-authors.js

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env node
+
+/*
+ * Generate an AUTHOR file on the repo root and on each lerna package based on git log.
+ *
+ * Add / change the ALIASES map to avoid duplications and show the correct names / emails.
+ *
+ * NOTE: Author lines with `users.noreply.github.com` emails are removed.
+ */
+
+const ALIASES = {
+  'Anna Henningsen <anna.henningsen@mongodb.com>': 'Anna Henningsen <addaleax@gmail.com>',
+  'Anna Henningsen <anna@addaleax.net>': 'Anna Henningsen <addaleax@gmail.com>',
+  'Anna Henningsen <github@addaleax.net>': 'Anna Henningsen <addaleax@gmail.com>',
```

Comment:
```suggestion
  'Anna Henningsen <anna.henningsen@mongodb.com>': 'Anna Henningsen <anna@addaleax.net>',
  'Anna Henningsen <addaleax@gmail.com>': 'Anna Henningsen <anna@addaleax.net>',
  'Anna Henningsen <github@addaleax.net>': 'Anna Henningsen <anna@addaleax.net>',
```

(or just use the mongob one, that’s also just fine)